apps/web/src/hooks/use-is-mobile.ts

```diff
@@ -3,7 +3,6 @@
 import { useLayoutEffect, useState } from 'react';
 
 function useMaxWidth(maxWidth: number) {
-  if (typeof window === 'undefined') false;
```

Comment:
this should be reverted
give early return on server

Reply:
1. It was not returning `false` at current state.
2. Early returns are not allowed before you define hooks. So that it the only place we can correctly put it

Reply:
it was working before, no?

Reply:
yeah because we bypassed the issue entirely